gtclang/cmake/GTClangConfig.cmake.in

```diff
@@ -8,8 +8,6 @@ message(STATUS "Found @PROJECT_NAME@ ${@PROJECT_NAME@_VERSION} at ${CMAKE_CURREN
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR})
 
 include(CMakeFindDependencyMacro)
-find_dependency(Clang 6.0.0 REQUIRED)
-
```

Comment:
Error is thrown if both `dawn` and `gtclang` do `find_dependency (Clang 10.0.0 EXACT REQUIRED)` in the cmake file. So I only do it in `dawn`, `gtclang` then inherits this.